index.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
+
+const NR = require('node-resque');
+const asCallback = require('ascallback');
+const config = require('config');
+const ecosystem = config.get('ecosystem');
+const executorConfig = config.get('executor');
+const ExecutorPlugin = require('screwdriver-executor-router');
+const executorPlugins = Object.keys(executorConfig).reduce((aggregator, keyName) => {
+    if (keyName !== 'plugin') {
+        aggregator.push(Object.assign({
+            name: keyName
+        }, executorConfig[keyName]));
+    }
+
+    return aggregator;
+}, []);
+const executor = new ExecutorPlugin({
+    defaultPlugin: executorConfig.plugin,
+    executor: executorPlugins,
+    ecosystem
+});
+const connectionDetails = {
+    pkg: 'ioredis',
+    host: process.env.REDIS_HOST,
+    password: process.env.REDIS_PASSWORD,
+    port: process.env.REDIS_PORT,
+    database: 0
+};
+
+const jobs = {
+    start: {
+        /**
+         * Call executor.start with the buildConfig
+         * @method perform
+         * @param {Object}  buildConfig               Configuration
+         * @param {Object}  [buildConfig.annotations] Optional key/value object
+         * @param {String}  buildConfig.apiUri        Screwdriver's API
+         * @param {String}  buildConfig.buildId       Unique ID for a build
+         * @param {String}  buildConfig.container     Container for the build to run in
+         * @param {String}  buildConfig.token         JWT to act on behalf of the build
+         */
+        perform: (buildConfig, callback) =>
+            asCallback(executor.start(buildConfig), (err) => {
```

Comment:
Can we look into why it sometimes didnt work, rather than introducing a library just to mask it? It could be that executor.start() is throwing exception ?